### DISCUSSION TITLE
Login to account and mobile app 2fa - failures

### DISCUSSION THREAD

**Discussion by datafiend** [community/community#38430] [Mobile] [answered]
Hi all,
I have been trying to login to github all morning. My 2fa is the github mobile app. I will login to desktop and the desktop will tell me to enter the digits onto the 2fa mobile app. 
I keep getting "failed to authenticate" - over and over again.
I've used several of my recovery codes(I couldn't believe i found them on my pc). 
So, I unistalled and re-installed the app and still doesn't work. 

Any ideas here? I'm going to run out of recovery codes soon. 
Thanks,

**Comment by airtower-luna** [comment]
I can't comment on the GitHub Mobile app, but before you run out of recovery codes, you can generate new ones. In the authentication settings you can view your current recovery codes, and if you do there's a button beneath them to generate new ones. I know that's not really a solution, but I think making sure you don't get locked out is most important now. :sweat_smile:

**Reply by datafiend** [reply]
ok. I still have about 6 or 7 left. I'll monitor my logins. 
Weird, but I just changed my windows login(for my work account) the other day and I'm getting A LOT OF login errors on other sites. Makes me wonder if this is a Microsoft authentication error?

**Comment by datafiend** [comment, marked as answer]
I logged out for an hour of the mobile app, now I logged in. It works. 

Sheesh.

**Comment by duldulao207** [comment]
Why, as of 2024-Sept-28, is v2rayN app not working on desktop, but is working on mobile phone?
